### PR TITLE
CircleCI - run nightly UAT tests

### PR DESCRIPTION
**Pull request by dincho**
Tests was ran 00:00 UTC last night: https://circleci.com/workflow-run/140fb9a6-eb3a-4ef2-bc97-7f6c287ede74 (indeterminate ?)

Successful builds can be found here:
- https://circleci.com/gh/aeternity/epoch/1542
- https://circleci.com/gh/aeternity/epoch/1527

However, I had to increase the test timeouts in 3e0e94df1244a1711af67a550c4aec44efab9c82 to make it work in CircleCI

### CODE REVIEW

**Comment by dincho** [comment]
I'll rework the commit history before merging.

File: .circleci/config.yml [hunk 1]
```diff
@@ -180,6 +180,33 @@ jobs:
...
+    <<: *container_config
+    steps:
+      - checkout
+      - *restore_rebar_cache
+      - *restore_build_cache
+      - run:
+          name: UAT Tests
+          command: |
+            epmd -daemon
+            make python-env && make swagger && make multi-build
```
Comment (lucafavatella) [inline .circleci/config.yml]: Why `make swagger`?

File: .circleci/config.yml [hunk 2]
```diff
@@ -180,6 +180,33 @@ jobs:
...
+            make python-uats
```
Comment (lucafavatella) [inline .circleci/config.yml]: Who does the start of the nodes? I would expect a `make multi-start` somewhere or equivalent.

File: .circleci/config.yml (hunk 1 above)
Comment (dincho) [reply .circleci/config.yml]: @lucafavatella I followed the docs here: https://github.com/aeternity/epoch/tree/master/py

If the docs are wrong, let's update them in this PR too.

File: .circleci/config.yml (hunk 2 above)
Comment (dincho) [reply .circleci/config.yml]: See above comment

Comment (dincho) [reply .circleci/config.yml]: @velzevur can you bring some light here?

Comment (velzevur) [reply .circleci/config.yml]: In each test we prepare a specific sys.config and start nodes using it. Nodes shouldn't be running before starting the tests.

Comment (lucafavatella) [inline .circleci/config.yml]: Are [ws-related python tests](https://github.com/aeternity/epoch/blob/84a9bef744e329f764e1b68966cbc0dfde523512/Makefile#L144) meant to be run too? Deleted?
Cc @velzevur

**Pull request merged**